### DISCUSSION TITLE
txt2img and img2img Workflows

### DISCUSSION THREAD

**Discussion by nic410826** [Comfy-Org/ComfyUI#6265] [Q&A]
Hello.
I am pretty new to AI and ComfyUI and would like to get your assistance for the following two 'issues':

1. I would like to formulate a positive prompt in order to create with a txt2img worflow a picture with e.g. three disdinguished persons (i.e. one person with white hairs, one with blond hairs and one with black hairs). I tried different prompts with different formulations, interpunction etc. but some of the characteristics of the three persons on the picture are generated identically (e.g. smiling, red lips etc). How do I formulate a prompt correctly so it will generate a picture according to my specifications? 

2. I would like to use an existing photograph which contains certain things I would like to integrate in a newly generated picture.  The source picture is only integrated into my target picture if I am reducing the Denoise value to less than 0.5; and in a very bad quality and not positioned correctly at the place where the source picture is showing it. The process does not work at all in a txt2img workflow with the embedding function.; apparently, the embedding function is not carried out at all... 

Obviously I am doing something wrong, but I don't know how to correct my error!

Thanks for feedbac!

Regards - Nicola (nic410826)

**Comment by ltdrdata** [comment]
1. The difference in generation capability varies greatly depending on the model's ability. Models like **SD1.5** and **SDXL** often experience a phenomenon called **prompt bleeding** and struggle to process long prompts effectively.  
With newer models like **SD3.5** and **FLUX**, prompts can be clearly specified and distinguished through explicit instructions.  

2. For this issue, check out a **ipadapter**.
   * https://github.com/cubiq/ComfyUI_IPAdapter_plus
   * https://www.youtube.com/@latentvision

**Reply by nic410826** [reply]
I will do so; thank you very much!
